lib/member_page.rb

```diff
@@ -36,6 +40,14 @@ class MemberPage < Scraped::HTML
 
   private
 
+  def titles
+    %w(Dr)
+  end
+
+  def name_parts
```

Comment:
There's really two different things going on in one method here — one is getting the text of the 'raw' name field — the other is processing that into parts. Those should really be separate methods.